ipydb/magic.py

```diff
@@ -128,7 +128,12 @@ def rollback(self, arg):
               help='pretty-print sql statement and exit')
     @argument('-o', '--output', action='store', dest='file',
               help='Write sql output as CSV to the given file')
+    @argument('-d', '--debug', action='store_true',
```

Comment:
I don't think the --debug flag is necessary. You can use the %debug and %pdb magic commands that come with ipython to achieve the same result (post-mortem debug and set breakpoints etc). The same applies for lines 177:179